internal/util/vault_tokens.go

```diff
@@ -50,6 +51,73 @@ const (
 	vaultTokenSecretKey = "token"
 )
 
+type standardVault struct {
+	KmsPROVIDER        string `json:"KMS_PROVIDER"`
```

Comment:
is it a problem if some of these keys are not set in the ConfigMap? Maybe they need `omitempty`?

Reply:
as we are unmarshaling it should be fine, omitEmpty is useful when  we are marshaling the data.